lib/ifupdown.sh.in

```diff
@@ -29,10 +40,10 @@ esac
 
 # Previous work (create the interface)
 if [ "$MODE" = 'start' ] && [ ! -d "/sys/class/net/$IFACE" ]; then
-  brctl addbr "$IFACE" || exit 1
+  "$brctl" addbr "$IFACE" || exit 2
```

Comment:
curious: why `exit 2` ?
i was not gonna ask initially, since `2` seems to be an error code implying a higher error state

but in commit 58468163ac6f320fafe80fe0168c37624cf8e470  you're using `exit 1` in `lib/mstp_config_bridge.in `